lightning/pkg/server/lightning.go

```diff
@@ -567,7 +567,7 @@ func (l *Lightning) run(taskCtx context.Context, taskCfg *config.Config, o *opti
 		if keyspaceName == "" {
 			keyspaceName, err = getKeyspaceName(db)
 			if err != nil {
-				o.logger.Warn("unable to get keyspace name, lightning will use empty keyspace name", zap.Error(err))
+				o.logger.Debug("unable to get keyspace name, lightning will use empty keyspace name", zap.Error(err))
```

Comment:
```suggestion
				o.logger.Info("unable to get keyspace name, lightning will use empty keyspace name", zap.Error(err))
```
maybe change to info, so we can know what's going on.

Reply:
there's another log in line 573

Reply:
The user will still see the "access denied" message from `zap.Error(err)` though